platform-hub-web/src/app/announcements/editor/announcements-editor-list.html

```diff
@@ -102,12 +102,12 @@
           <span md-colors="{color: 'blue-grey-700'}">
             Created at:
           </span>
-          <span>{{a.created_at}}</span>
+          <span>{{a.created_at | date:'EEE MMM d y HH:mm UTC':'UTC'}}</span>
```

Comment:
We do use `medium` as the date format in the user's list, however that doesn't include the timezone and it doesn't normalise the time to UTC (nor does it include the day of the week), so I've started using this custom format in recent features. At some point we should probably consolidate and use the same format every (wrapping it in a custom filter to make it nicer to use).